content/pt-br/docs/tasks/configure-pod-container/assign-cpu-resource.md

```diff
@@ -0,0 +1,270 @@
+
+{{< codenew file="pods/resource/cpu-request-limit-2.yaml" >}}
+
+
+Crie o Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/pods/resource/cpu-request-limit-2.yaml --namespace=cpu-example
+```
+
+Veja o status do Pod:
+
+
+```shell
+kubectl get pod cpu-demo-2 --namespace=cpu-example
+```
+
+A saída mostra que o status do Pod é `Pending`, demonstrando que o Pod não está 
+agendado para executar em qualquer Nó, e permanecerá no estado Pendente indefinidamente.
+
+
+```
+NAME         READY     STATUS    RESTARTS   AGE
+cpu-demo-2   0/1       Pending   0          7m
+```
+
+Para ver informações detalhadas sobre o Pod, incluindo eventos:
+
+
+```shell
+kubectl describe pod cpu-demo-2 --namespace=cpu-example
+```
+
+A saída mostra que o Contêiner não pode ser agendado devido a insuficiência 
+de recursos de CPU nos Nós: 
+
+
+```
+Events:
+  Reason                        Message
+  ------                        -------
+  FailedScheduling      No nodes are available that match all of the following predicates:: Insufficient cpu (3).
+```
+
+Apague seu Pod:
+
+```shell
+kubectl delete pod cpu-demo-2 --namespace=cpu-example
+```
+
+## Se você não especificar um limite de CPU
+
+Se você não especificar o limite de CPU para um Contêiner, então uma destas 
+situações se aplica:
+
+* O Contêiner não possui limite superior de recursos de CPU que possa usar. 
+O Contêiner poderia usar todos os recursos de CPU disponíveis no Nó em que estiver executando. 
+
+
+* O Contêiner está executando em um namespace que tem um limite de CPU padrão, 
+e ao Contêiner é atribuído o limite padrão automaticamente. 
+Administradores de cluster podem usar [LimitRange](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#limitrange-v1-core/) 
+para especificar o valor padrão do limite de CPU.
+
+
+## Se você especificar um limite de CPU mas não especificar o requisito de CPU
+
+
+Se você especificar o limite de CPU para um Contêiner mas não especificar 
+os requisitos de CPU, o Kubernetes automaticamente irá atribuir os requisitos 
+de CPU que correspondem ao limite. Similarmente, se o Contêiner especifica 
+seu próprio limite de CPU mas não especifica os requisitos de CPU, 
+o kubernetes automáticamente atribui os requisitos de CPU que correspondem ao limite.
+
+## Motivos para usar requisitos e limites de CPU
+
+Ao configurar os requisitos de CPU e limites dos Contêneres que executam no seu cluster, 
+você faz um uso eficiente dos recursos de CPU disponívies nos Nós do seu cluster. 
+Mantendo baixos os recursos de CPU de um Pod, você proporciona ao Pod uma boa chance 
+de ser agendado. Ao especificar um limite de CPU que é maior que os requisitos de CPU, 
+você estará realizando duas coisas: 
+
+* o Pod pode ter picos de atividade quando estiver fazendo uso de recursos de CPU 
```

Comment:
Minor: add capital "O" here in the beginning of the sentence for consistency.

Original documentation says "where", but that was translated to "quando". Suggestion: "picos de atividade onde utilizará recursos de CPU que estão disponíveis" delivers that better.